Sources/Plasma/NucleusLib/pnAsyncCoreExe/pnAceSocket.cpp

```diff
@@ -116,6 +116,7 @@ struct ConnectOperation
     void*                   fParam;
     std::vector<uint8_t>    fConnectBuffer;
     unsigned int            fConnectionType;
+    std::recursive_mutex    critsect;

```

Comment:
Needs the f prefix

Reply:
I think this is more dead code from the timer/soft disconnect path. Removing.